tools_and_services/preferred_services.md

```diff
@@ -1,15 +1,36 @@
 # Preferred Services
 
-## Mailing
-[SendGrid](https://sendgrid.com/), in Heroku: use the free Starter addon (Automatically added by [Muffi](https://github.com/abtion/muffi)).
+Finding a balance between trying out a new service or using a well-known tool is tricky. There are several things to consider: can a tool that we previously used solve the problem? Is the learning curve of the new tool justified by the gained efficiency? Is the new (or old tool) GDPR compliant?
+As the matter is tricky, simply listing previously used tools is not enough. Instead, we have a three-color system.
+
+
+- ![](https://via.placeholder.com/10/00c900/?text=+) Green list: These services have been widely used at Abtion, and they usually provide the necessary functionality that we expect from them.
+If there is a service in this category, but the team wants to use a different one, the team must first check with the Principal Engineer or CTO.
+- ![](https://via.placeholder.com/10/ffff00/?text=+) Yellow list: We've used the services before, and we are satisfied. The services in this list should be the first option to consider on a team, although the team can decide to use a different tool if there is a good reason behind it.
```

Comment:
Shouldn't it rather be the services in the green list that are the first options to consider?